platform/darwin/src/MGLLogging.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+#import "MGLFoundation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSInteger, MGLLoggingLevel) {
+    
+    MGLLoggingLevelNone = 0,
+    
+    MGLLoggingLevelInfo,
+    
+    MGLLoggingLevelDebug,
+    
+    MGLLoggingLevelError,
+    
+    MGLLoggingLevelFault,
+    
+};
+
+typedef void (^BlockHandler)(MGLLoggingLevel, NSString *, NSString *);
+
+MGL_EXPORT
+@interface MGLLogging : NSObject
+
+@property (nonatomic, copy, null_resettable) BlockHandler handler;
+@property (assign, nonatomic) MGLLoggingLevel loggingLevel;
```

Comment:
I don't understand what you mean. The way is designed is that it will provide a default logging block and you can configure the logging level in the same class.